alert-policies/ray/ActiveNodes.yml

```diff
@@ -0,0 +1,31 @@
+# Name of the alert
+name: Ray Active Nodes
+
+# Description and details
+description: |+
+  This alert is triggered if the active nodes lesser than 1 for 5 minutes.
```

Comment:
Unclear what is the meaning of this

Reply:
Hi @seemantk,
I have rephrased the description. Could you please review it?